runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1558,135 +902,151 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadArgMax(const Operator *op, ir
-      loadConv2D(op, subg);
+    {
+      ir::operation::Conv2D::Param param;
+      const auto *options = op->builtin_options_as_Conv2DOptions();
+      param.activation = convertActivation(options->fused_activation_function());
+      loadStridesAndPaddings(param, options);
+
+      param.dilation.width_factor = options->dilation_w_factor();
+      param.dilation.height_factor = options->dilation_h_factor();
+
+      loadOperationTo<ir::operation::Conv2D>(op, subg, param);
       return;
+    }
     case BuiltinOperator::BuiltinOperator_AVERAGE_POOL_2D:
       loadPool2D(op, subg, ir::operation::Pool2D::PoolType::AVG);
       return;
+    case BuiltinOperator::BuiltinOperator_MAX_POOL_2D:
+      loadPool2D(op, subg, ir::operation::Pool2D::PoolType::MAX);
+      return;
     case BuiltinOperator::BuiltinOperator_DEPTHWISE_CONV_2D:
-      loadDepthwiseConv2D(op, subg);
+    {
+      ir::operation::DepthwiseConv2D::Param param;
+      const auto *options = op->builtin_options_as_DepthwiseConv2DOptions();
+      param.activation = convertActivation(options->fused_activation_function());
+      loadStridesAndPaddings(param, options);
+      // Multiplier
+      param.multiplier = options->depth_multiplier();
+
+      loadOperationTo<ir::operation::DepthwiseConv2D>(op, subg, param);
       return;
+    }
     case BuiltinOperator::BuiltinOperator_TRANSPOSE_CONV:
-      loadTransposeConv(op, subg);
+    {
+      ir::operation::TransposeConv::Param param;
+      const auto *options = op->builtin_options_as_TransposeConvOptions();
+      loadStridesAndPaddings(param, options);
+
+      loadOperationTo<ir::operation::TransposeConv>(op, subg, param);
       return;
+    }
     case BuiltinOperator::BuiltinOperator_RESHAPE:
-      loadReshape(op, subg);
+    {
+      ir::operation::Reshape::Param param;
+      const auto *options = op->builtin_options_as_ReshapeOptions();
+      if (options != nullptr)
+      {
+        const auto *new_shape = options->new_shape();
+        if (new_shape)
+        {
+          for (uint i = 0; i < new_shape->Length(); ++i)
+          {
+            param.new_shape.push_back(new_shape->Get(i));
+          }
+        }
+      }
+      loadOperationTo<ir::operation::Reshape>(op, subg, param);
       return;
+    }
     case BuiltinOperator::BuiltinOperator_SOFTMAX:
-      loadSoftmax(op, subg);
-      return;
-    case BuiltinOperator::BuiltinOperator_MAX_POOL_2D:
-      loadPool2D(op, subg, ir::operation::Pool2D::PoolType::MAX);
+    {
+      ir::operation::Softmax::Param param;
+      const auto *options = op->builtin_options_as_SoftmaxOptions();
+      // Beta
+      param.beta = options->beta();
+      loadOperationTo<ir::operation::Softmax>(op, subg, param);
       return;
+    }
     case BuiltinOperator::BuiltinOperator_CONCATENATION:
-      loadConcatenation(op, subg);
+    {
+      ir::operation::Concat::Param param;
+      const auto *options = op->builtin_options_as_ConcatenationOptions();
+      // Axis
```

Comment:
I understand that this is cut & paste, but as it is a refactoring, I would suggest to remove such obvious comments.

Reply:
Okay I will remove it.